Improve potential reasons for prometheus load increase

idavidmcdonald opened the pull request:
Just a small improvement I thought could be useful based on some of the previous situations we've seen for prometheus struggling under load.

philandstuff commented on source/documentation/observe-support.md:
```diff
@@ -135,9 +135,13 @@ For more information you could [search the prometheus-aws-configuration-beta rep
+- queries being run which are too resource intensive as they query over too many metrics or too long a time period
+- an increase in the number of metrics being scraped causing existing queries to be too resource intensive
+
+Queries can originate from a Grafana instance or be manually run by a user.
```
or alerting or recording rules

philandstuff commented on source/documentation/observe-support.md:
```diff
@@ -147,8 +151,13 @@ For more information you could [search the prometheus-aws-configuration-beta rep
+- queries being run which are too resource intensive as they query over too many metrics or too long a time period
+- an increase in the number of metrics being scraped causing existing queries to be too resource intensive
+
+Queries can originate from a Grafana instance or be manually run by a user.
```
I can't tell from these runbooks that `RE_Observe_Prometheus_Over_Capacity` is the 'major 💩-storm' version and `RE_Observe_Prometheus_High_Load` is the 'mild peril' version.  They both use the phrase "critically over capacity" but I think this is somewhat misleading.

idavidmcdonald commented:
Think I've addressed your comments @philandstuff. I agree that the alert names could be a bit more distinct but I'm going to avoid changing them and having to do a deploy for the moment just so I can get this in.
